src/ComServerHelpers/ComServerHelpers.csproj

```diff
@@ -0,0 +1,79 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <RootNamespace>ComServerHelpers</RootNamespace>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DebugType>full</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <Authors>Microsoft Corporation</Authors>
+    <Company>Microsoft Corporation</Company>
+    <Product>C#/WinRT</Product>
+    <PackageId>ComServerHelpers</PackageId>
+    <FileVersion>$(VersionNumber)</FileVersion>
+    <Version>$(VersionNumber)</Version>
```

Comment:
Wonder if we should adopt our own assembly version for this binary rather than use the WinRT.Runtime one.

Reply:
Up to you :)
If you want a separate version - I'll set it up

Reply:
Yea we should probably do separate versioning to reduce friction when we add new APIs here.

Reply:
Added separate versioning.